utils/checkdb.py

```diff
@@ -64,35 +65,57 @@ def check_tile_overlap(db, db_root, strict=False, verbose=False):
     Throw an exception if two tiles share an address
     '''
     mall = dict()
+    tiles_type_done = dict()
+    tile_segbits = dict()
 
     tiles_checked = 0
 
-    def subtiles(tile_names):
-        for tile_name in tile_names:
-            yield tile_name, db.tilegrid[tile_name]
-
     for tile_name, tilej in db.tilegrid.items():
```

Comment:
Use db.grid() rather than directly use the underlying data.

Reply:
Done